### DISCUSSION TITLE
[QST] Why were TPC Benchmarks Removed?

### DISCUSSION THREAD

**Discussion by barnes88** [NVIDIA/spark-rapids#5365] [General] [answered]
Recently the TPC benchmarks were [removed from the project repository](https://github.com/NVIDIA/spark-rapids/pull/1826). Why were the benchmarks and results removed? Are there any major modifications that were made to the tpc(like) benchmarks that were in this repo compared to official benchmark query templates or a framework like [this one from databricks?](https://github.com/databricks/spark-sql-perf)

**Comment by sameerz** [comment, marked as answer]
Hi @barnes88, thanks for the question.  We are still using industry standard benchmarks for this project.  We felt it best to keep the benchmarks outside of the repository for the time being and are considering where best to host them.  We did not make any major modifications compared to the queries that are in the databricks repository.

**Reply by GaryShen2008** [reply]
Hi @barnes88 , we published a new open-source project https://github.com/NVIDIA/spark-rapids-benchmarks/. You can find the subfolder nds, which is based on TPCDS. We support data generation, query generation, power run and throughput run. Please refer to the [readme](https://github.com/NVIDIA/spark-rapids-benchmarks/tree/branch-22.06/nds).

**Reply by barnes88** [reply]
Great! Thank you for the update.
